packages/components/src/range-control/input-range.js

```diff
@@ -23,23 +34,30 @@ function InputRange(
 		label,
 		onHideTooltip = noop,
 		onMouseLeave = noop,
-		step = 1,
 		onBlur = noop,
 		onChange = noop,
 		onFocus = noop,
 		onMouseMove = noop,
+		onShiftStep,
 		onShowTooltip = noop,
-		shiftStep = 10,
+		shiftStep,
```

Comment:
What is the reason for removing the default value of `shiftStep`? Is it because a default value is already assigned on the parent `RangeControl` component? Or is there another reason?

Reply:
> Is it because a default value is already assigned on the parent RangeControl component?

🎯 Exactly that.